frontend/src/components/input-form/input-form.jsx

```diff
@@ -234,7 +234,7 @@ class InputForm extends React.Component {
       const pgType = this.state.pageType
       return function(e) {
         e.preventDefault();
-        if(buttonName === 'Login' || buttonName === 'Modify Profile' || (buttonName === 'Sign Up' && pgType === 'SIGNUP') || (pgType === 'ADD-LISTING')) { 
+        if(buttonName === 'Login' || buttonName === 'Modify Profile'|| (buttonName === 'Reset' && pgType === 'FORGOT-PASSWORD') || (buttonName === 'Sign Up' && pgType === 'SIGNUP') || (pgType === 'ADD-LISTING')) { 
```

Comment:
you need to put this in a constant variable with a good name because nobody has any indication of what this is supposed to be checking for as it is